micall/core/aln2counts.py

```diff
@@ -1022,33 +1022,44 @@ def write_counts(self,
                 break
 
     def merge_extra_counts(self):
-        for region, report_aminos in self.reports.items():
+        for region in self.report_nucleotides:
```

Comment:
These dictionaries have fewer than 100 entries, so it's not likely to be a significant performance difference, but as a good habit, I try to use `for key, value in mydict.items():` instead of `for key in mydict:` followed by `value = mydict[key]`.

Reply:
Thanks for the reminder, this is definitely a habit I should build.